src/content/learn/react-developer-tools.md

```diff
@@ -1,33 +1,34 @@
 ---
-title: React Developer Tools
+title: Alat pengembang React
 ---
 
 <Intro>
 
-Use React Developer Tools to inspect React [components](/learn/your-first-component), edit [props](/learn/passing-props-to-a-component) and [state](/learn/state-a-components-memory), and identify performance problems.
+Gunakan Alat Pengembang React untuk memeriksa [komponen](/learn/your-first-component), menyunting [*props*](/learn/passing-props-to-a-component) dan [*state*](/learn/state-a-components-memory), dan mengidentifikasi masalah kinerja.
 
 </Intro>
 
 <YouWillLearn>
 
-* How to install React Developer Tools
+* Cara memasang Alat Pengembang React
```

Comment:
```suggestion
* Cara menginstal React Developer Tools
```

install -> instal

Please check the [Glossary](https://github.com/reactjs/id.react.dev/wiki/Glosarium).